Orange/widgets/visualize/owboxplot.py

```diff
@@ -816,6 +818,15 @@ def draw_axis(self):
         self.box_scene.addLine(
             bottom * scale_x - 4, 0, top * scale_x + 4, 0, self._pen_axis)
 
+    def draw_stat(self):
+        if self.stat_test:
+            l = QGraphicsSimpleTextItem(self.stat_test)
+            l.setFont(self._axis_font)
```

Comment:
The text does not render correctly on my machine -- small parentheses, and no `'` in "Student's".

<img width="218" alt="Screenshot 2019-07-31 at 16 10 50" src="https://user-images.githubusercontent.com/2387315/62219094-f6a17000-b3ad-11e9-8e86-554f9386c03d.png">

It works (and also looks OK) if I remove `setFont`.

Reply:
Our `_axis_font`, which is just the default (`_axis_font = QFont()`), doesn't display some characters nicely? This is bad.